py/torch_tensorrt/__init__.py

```diff
@@ -94,7 +94,11 @@ def _find_lib(name, paths):
 
 from torch_tensorrt import fx
 
-if version.parse(torch.__version__) >= version.parse("2.1.dev"):
+if version.parse(
+    torch.__version__
```

Comment:
Make a little macro function in utils that just returns the cleaned torch version

Reply:
Could this function instead go in `py/torch_tensorrt/_version.py`, to avoid the `from torch_tensorrt import _C` which occurs in the `py/torch_tensorrt/_util.py` file?

Reply:
_version.py gets generated by setup.py so probably not a great choice, but i dont really care where it goes, could go in `__init__.py` or somewhere else

Reply:
Alternatively, we probably should be moving _C dep code into .ts anyway. Half the functions could probably be reimplemented without _C

Reply:
https://github.com/pytorch/TensorRT/blob/d4c9c06213602c6e8d2ce568154ed287ee67dafd/py/torch_tensorrt/_util.py#L12C5-L12C19 This code just references https://github.com/pytorch/TensorRT/blob/main/core/util/build_info.h

Reply:
I think lets put it in _util.py with an AI to move the _C code out pre 2.1

Reply:
@gs-olive add a task to do this in https://github.com/pytorch/TensorRT/issues/2119

Reply:
This makes sense, I can add the task. I did some minor refactoring of `_util.py` in this PR since it was a minimal change. Let me know if this change is within the scope of the PR/is favorable.

Reply:
Added #2121 and rolled back `_util.py` refactoring changes initially made in 38a2c4cf1694d39d5cf0e17df4db07300d0955fd, since they are out of scope for this PR.